now.json

```diff
@@ -5,5 +5,6 @@
         "node": "10.x.x"
     },
     "public": true,
-    "version": 1
+    "version": 1,
+    "type": "npm"
```

Comment:
because a `Dockerfile` is present `now` will automatically try to do a docker deploy, but I didn't set this up for that purpose, but I can look into it if needed.

Reply:
Thanks for catching this. Yeah, we don't want to invest too much in refactoring our Now presence, so this is the right thing to do.